examples/thoughts/ext/MainPage.js

```diff
@@ -124,12 +124,10 @@ const Thought = (props) => {
 
         <div className="thought-tags">
           { tagNames.map(tagName => (
-            <div className="thought-tags-tag"
-                 onClick={() => removeTag(tagName)}
-                 key={tagName}
```

Comment:
I wonder, doesn't React complain since you removed the `key` prop?

Reply:
Yes it does, added this prop back.